ui/app/package.json

```diff
@@ -185,16 +185,19 @@
     "@types/prettier": "^3.0.0",
     "@types/react-infinite-scroller": "^1.2.5",
     "@types/zxcvbn": "^4.4.4",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
     "babel-plugin-formatjs": "^10.5.10",
     "babel-plugin-react-intl": "^8.2.25",
     "build-if-changed": "^1.5.5",
     "eslint": "^8.55.0",
     "eslint-config-react-app": "^7.0.1",
+    "fork-ts-checker-webpack-plugin": "^6.5.3",
```

Comment:
Why the new packages?

Reply:
same as https://github.com/craftercms/studio-ui/pull/3516#discussion_r1424126783